Riot/Modules/Room/Views/WYSIWYGInputToolbar/WysiwygInputToolbarView.swift

```diff
@@ -101,7 +101,9 @@ class WysiwygInputToolbarView: MXKRoomInputToolbarView, NibLoadable, HtmlRoomInp
         }, showSendMediaActions: { [weak self]  in
             guard let self = self else { return }
             self.showSendMediaActions()
-        })
+        }).introspectTextView { textView in
+            textView.inputAccessoryView = self.inputAccessoryViewForKeyboard
```

Comment:
I see that this function takes an `@escaping` closure, maybe would be safer to have either a `weak` or `unowned self`?

Reply:
ah yes I can planned on doing that and never did, thanks!